cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectDockerfile.java

```diff
@@ -284,6 +357,47 @@ private void addZipArtifact(PrintWriter writer, String connectorPath, ZipArtifac
         writer.println();
     }
 
+    /**
+     * Add command sequence for downloading Maven artifact
+     *
+     * @param writer            Writer for printing the Docker commands
+     * @param connectorPath     Path where the connector to which this artifact belongs should be downloaded
+     * @param mvn               The maven artifact
+     */
+    private void addMavenArtifact(PrintWriter writer, String connectorPath, MavenArtifact mvn) {
+        validateGavPresence(mvn);
+        String repo = mvn.getRepository() == null ? MavenArtifact.DEFAULT_REPOSITORY : maybePatchRepository(mvn.getRepository());
+        String artifactHash = Util.sha1Prefix(mvn.getGroup() + mvn.getArtifact() + mvn.getVersion());
+        String artifactDir = connectorPath + "/" + artifactHash;
+
+        String downloadJarCmd = String.format("curl -L --create-dirs --output %s/%s-%s.jar %s%s/%s/%s/%s-%s.jar",
```

Comment:
Why do we have to download the JAR? Isn't it downloaded in the pre-step? If we have to do it, wouldn't it be better to do it in the pre-stage as well? It would save us step / layer in the final image.